apps/provider-proxy/test/setup/providerServer.ts

```
@@ -98,8 +67,10 @@ export function stopProviderServer() {
   runningServer?.close();
 }
 
+type RequestHandlers = Record<string, (req: IncomingMessage, res: ServerResponse) => (() => void) | undefined | void>;
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)